src/components/HypothesisTesting/NormalCurve.js

```diff
@@ -64,7 +64,7 @@ export default function NormalCurve({ means, mu0, popStandardDev, sampleSize, di
     means.forEach(({ testStatistic, mean, reject }) => {
       meanCounts[mean] = _.defaultTo(meanCounts[mean] + 1, 1);
       const meanObject = {
-        x: mean,
+        x: testType === "oneSample" ? mean : meansDiff,
```

Comment:
Note that the `mean` attribute already holds the difference of means for a two-sample test. We don't need to pass an additional prop here.